hazelcast/src/main/java/com/hazelcast/instance/Node.java

```diff
@@ -124,6 +123,7 @@
 
     private final NodeExtension nodeExtension;
 
+    private final com.hazelcast.internal.properties.HazelcastProperties properties;
```

Comment:
Yes, because we have the deprecated class `HazelcastProperties` in the same package, which can be removed in 3.8.